decidim-core/lib/decidim/core/test/shared_examples/comments_examples.rb

```diff
@@ -64,7 +64,7 @@
       end
 
       it "shows comment to the user" do
-        expect(page).to have_comment_from(user, "This is a new comment")
+        expect(page).to have_comment_from(user, "This is a new comment", wait: 20)
```

Comment:
@mrcasals this is **max** wait time, correct?

Reply:
That's correct!